plugin/core/sessions.py

```diff
@@ -1343,12 +1347,13 @@ def update_folders(self, folders: List[WorkspaceFolder]) -> None:
         if self.should_notify_did_change_workspace_folders():
             added, removed = diff(self._workspace_folders, folders)
             if added or removed:
+                change_event = {
+                    "added": [a.to_lsp() for a in added],
+                    "removed": [r.to_lsp() for r in removed]
+                }  # type: WorkspaceFoldersChangeEvent
                 params = {
-                    "event": {
-                        "added": [a.to_lsp() for a in added],
-                        "removed": [r.to_lsp() for r in removed]
-                    }
-                }
+                    "event": change_event
+                }  # type: DidChangeWorkspaceFoldersParams
```

Comment:
Not an issue but I guess there was no technical with keeping it as a one variable?